### PR TITLE
add change log and bump version to 0.1.11

### PR DESCRIPTION
**Pull request by freeznet**
### Motivation

release v0.1.11

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

Check the box below.

Need to update docs? 

- [ ] `doc-required` 
  
  (If you need help on updating docs, create a doc issue)
  
- [ ] `no-need-doc` 
  
  (Please explain why)
  
- [ ] `doc` 
  
  (If this PR contains doc changes)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
+# Function Mesh v0.1.11 Release Notes
+
+## v0.1.11 What's New
+
+- [Helm Charts] support more controller manager params for helm charts ([#339](https://github.com/streamnative/function-mesh/pull/339))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:5]:
```suggestion
- [Helm Charts] Support more controller manager parameters for Helm charts ([#339](https://github.com/streamnative/function-mesh/pull/339))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,23 @@
...
+- [Helm Charts] fix helm charts install ([#347](https://github.com/streamnative/function-mesh/pull/347)), ([#346](https://github.com/streamnative/function-mesh/pull/346)), ([#345](https://github.com/streamnative/function-mesh/pull/345)), ([#344](https://github.com/streamnative/function-mesh/pull/344))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:6]:
```suggestion
- [Helm Charts] Fix Helm charts installation issues ([#347](https://github.com/streamnative/function-mesh/pull/347)), ([#346](https://github.com/streamnative/function-mesh/pull/346)), ([#345](https://github.com/streamnative/function-mesh/pull/345)), ([#344](https://github.com/streamnative/function-mesh/pull/344))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,23 @@
...
+- [Helm Charts] support namespace config for helm charts ([#327](https://github.com/streamnative/function-mesh/pull/327))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:7]:
```suggestion
- [Helm Charts] Support namespace configurations for Helm charts ([#327](https://github.com/streamnative/function-mesh/pull/327))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,23 @@
...
+- [Controller] apply Istio service port name convention to services ([#343](https://github.com/streamnative/function-mesh/pull/343))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:8]:
```suggestion
- [Controller] Apply the Istio service port name convention to services ([#343](https://github.com/streamnative/function-mesh/pull/343))
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,23 @@
...
+- [Controller] fix webhook service ([#341](https://github.com/streamnative/function-mesh/pull/341))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:9]:
```suggestion
- [Controller] Fix the Webhook service ([#341](https://github.com/streamnative/function-mesh/pull/341))
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,23 @@
...
+- [Controller] Move `apiextensions.k8s.io/v1beta1` to `apiextensions.k8s.io/v1` ([#328](https://github.com/streamnative/function-mesh/pull/328))
+- [Controller] fix service `controller-manager-metrics-service` wrong selector ([#333](https://github.com/streamnative/function-mesh/pull/333))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:11]:
```suggestion
- [Controller] Fix the incorrect selector for the `controller-manager-metrics-service` service ([#333](https://github.com/streamnative/function-mesh/pull/333))
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,23 @@
...
+- [Controller] expose more parameters for manager ([#329](https://github.com/streamnative/function-mesh/pull/329))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:12]:
```suggestion
- [Controller] Expose more parameters for the controller manager ([#329](https://github.com/streamnative/function-mesh/pull/329))
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,23 @@
...
+- [Controller] add controller config file ([#322](https://github.com/streamnative/function-mesh/pull/322))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:13]:
```suggestion
- [Controller] Add the controller configuration file ([#322](https://github.com/streamnative/function-mesh/pull/322))
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,3 +1,23 @@
...
+- [Controller] Add annotation to exclude the webhook port from Istio proxying ([#321](https://github.com/streamnative/function-mesh/pull/321))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:14]:
```suggestion
- [Controller] Add annotations to exclude the Webhook port from Istio proxying ([#321](https://github.com/streamnative/function-mesh/pull/321))
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,3 +1,23 @@
...
+- [OLM] Increase resource limit for manager ([#324](https://github.com/streamnative/function-mesh/pull/324))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:15]:
```suggestion
- [OLM] Increase the resource limit for the controller manager ([#324](https://github.com/streamnative/function-mesh/pull/324))
```

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,3 +1,23 @@
...
+- [Runner Images] verify Pulsar 2.10 ([#336](https://github.com/streamnative/function-mesh/pull/336))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:16]:
```suggestion
- [Runner Images] Verify Pulsar 2.10 ([#336](https://github.com/streamnative/function-mesh/pull/336))
```

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,3 +1,23 @@
...
+- [Runner Images] bump sample dependencies ([#317](https://github.com/streamnative/function-mesh/pull/317)), ([#318](https://github.com/streamnative/function-mesh/pull/318))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:17]:
```suggestion
- [Runner Images] Bump sample dependencies ([#317](https://github.com/streamnative/function-mesh/pull/317)), ([#318](https://github.com/streamnative/function-mesh/pull/318))
```

File: CHANGELOG.md [hunk 13]
```diff
@@ -1,3 +1,23 @@
...
+- [Test] Bump versions for test script toolsets ([#334](https://github.com/streamnative/function-mesh/pull/334))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:18]:
```suggestion
- [Test] Bump versions for the test script toolsets ([#334](https://github.com/streamnative/function-mesh/pull/334))
```

File: CHANGELOG.md [hunk 14]
```diff
@@ -1,3 +1,23 @@
...
+- [Test] Remove unused steps in test scripts ([#331](https://github.com/streamnative/function-mesh/pull/331))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:19]:
```suggestion
- [Test] Remove unused steps from the test scripts ([#331](https://github.com/streamnative/function-mesh/pull/331))
```

File: README.md [hunk 15]
```diff
@@ -20,6 +20,19 @@ Note:
 - [pulsar-cluster](https://pulsar.apache.org/docs/en/pulsar-2.0/)
 - [pulsar-functions](https://pulsar.apache.org/docs/en/functions-overview/)
 
+## Compatibility
+
+### Kubernetes compatibility matrix
+
+The following Kubernetes versions are supported and work as we test against these versions in their respective branches. But note that other versions might work as well.
```
Comment (Huanli-Meng) [inline README.md:27]:
```suggestion
This table outlines the supported Kubernetes versions. We have tested these versions in their respective branches. But note that other versions might work as well.
```

**Review by Huanli-Meng — COMMENTED** [review]
Leave my comments. PTAL. Thanks

**Review by nlu90 — APPROVED** [review]

**Pull request merged**